api/room/models.py

```diff
@@ -18,7 +18,7 @@ class Room(Base, Utility):
     location = Column(String, ForeignKey('locations.name'))
     floor_id = Column(Integer, ForeignKey('floors.id'))
     floor = relationship('Floor')
-    locations = relationship('Location')
+    location_name = relationship('Location')
```

Comment:
Any reason for specifically calling this 'location_name' and not location to reference the table Location?